icons/power-circle.svg

```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <circle cx="12" cy="12" r="10" />
   <path d="M12 12V6" />
-  <path d="M8 7.5A6.1 6.1 0 0 0 12 18a6 6 0 0 0 4-10.5" />
+  <path d="M8 7.5a6 6 0 1 0 8 0" />
+  <circle cx="12" cy="12" r="10" />
```

Comment:
I've also updated this in #1906, I'm pretty sure this should match `square-power`:

![image](https://github.com/lucide-icons/lucide/assets/17746067/b4da9d36-ca1d-4e2e-9556-431a8d112461)


```svg
  <circle cx="12" cy="12" r="10" />
  <path d="M12 12V7" />
  <path d="M16 9a5 5 0 1 1-8 0" />
```

Reply:
Yeah, that looks a lot better with the smaller one, let's get the rename PR merged and then do the change for you suggest here.